SimTracker/TrackAssociatorProducers/plugins/QuickTrackAssociatorByHitsImpl.h

```diff
@@ -5,6 +5,8 @@
 
 #include "SimDataFormats/Associations/interface/TrackToTrackingParticleAssociator.h"
 
+#include "SimTracker/TrackerHitAssociation/interface/TrackerHitAssociator.h"
```

Comment:
There should be no need for this include.

Reply:
ok, it was in 
https://github.com/makortel/cmssw/pull/8/files/06ffcd750fb4dbf19fd62b49af863943c1f1d914#diff-e9a6ac580c652ef2ceb2db680ab04f46L6
and I decided to move it here, but I can replace it with the class that I need here.

Reply:
Which class do you need here? `SiTrackerMultiRecHit`? If you move `getMostProbablyHit` to a free function you don't need even that.

Reply:
Yes, true. This is what I noticed in this moment.